setup/centos6/resources/jenkins.initd

```diff
@@ -21,19 +21,21 @@ PIDFILE=/var/run/jenkins/$NAME.pid
 # Read config
 [ -f "$JENKINS_CONFIG" ] && . "$JENKINS_CONFIG"
 
-# Add scl-specific path as well as regular path
-JENKINS_PATH="$JENKINS_PATH:$PATH"
+# Add scl-specific path as well as regular path.
+# We add /usr/local/bin since that's where ccache symlinks live
+JENKINS_PATH="/usr/local/bin:$JENKINS_PATH:$PATH"
```

Comment:
yes, invoking our ccache script installs/symlinks into `/usr/local/bin`, and I just switched the python symlink path from our custom `/home/iojs/bin/` to `/usr/local/bin`.